### PR TITLE
/api/create_user accepts unique emails and unique + valid sensorids

### PR DESCRIPTION
**Pull request by juliawoodcock**

### CODE REVIEW

File: backend/frontend_integration.py [hunk 1]
```diff
@@ -112,8 +112,37 @@ def create_user():
         sensorid = data.get('sensorid')
         print(sensorid)
 
-        timestamp = int(time.time() * 1000) 
-        userid = timestamp % 1000000 #id will be 6 digits long
+        try: # make sure the sensorid is correct!
+            db_user_id = db.session.query(NewSensor.userid).filter(NewSensor._id == sensorid).first()
+            if db_user_id is None:
+                return jsonify({'message': 'Hmmm, are you sure that you have the right SensorId?'}), HTTPStatus.UNAUTHORIZED
```
Comment (draymond63) [inline backend/frontend_integration.py:119]: Capitalization in the message doesn't match the specified parameter

File: backend/frontend_integration.py [hunk 2]
```diff
@@ -112,8 +112,37 @@ def create_user():
...
+        except:
+            print ("The query broke! Not ideal.")
+            return jsonify({'message': 'Something broke!'}), HTTPStatus.INTERNAL_SERVER_ERROR
```
Comment (draymond63) [inline backend/frontend_integration.py]: Better error message? We know something broke while querying the sensor table

File: backend/frontend_integration.py [hunk 3]
```diff
@@ -112,8 +112,37 @@ def create_user():
...
+            db_email = db.session.query(FakeUser.email).filter(FakeUser.email == email).first()
+            if db_email is None:
+                print("Unique email found. User can proceed.")
+            else:
+                print("I found this email!")
+                print(db_email[0])
+                return jsonify({'message': 'Hmmm, it seems you already have an account with that email!'}), HTTPStatus.UNAUTHORIZED
+
+        except:
+            print("Something broke!")
```
Comment (draymond63) [inline backend/frontend_integration.py:136]: If something broke here, should we continue? Right now it will

**Comment by juliawoodcock** [comment]
@draymond63 pushed some changes to above!

**Review by draymond63 — APPROVED** [review]

**Pull request merged**